Add a button to reset the list of TLEs to defaults compiled in Sdrangel

dforsi opened the pull request:
This PR adds a convenience feature for users experiencing issue #1957 and to also get the new TLE source added in PR #1954

srcejon commented on plugins/feature/satellitetracker/satellitetrackersettingsdialog.cpp:
#include "qmessagebox.h" should be #include <QMessageBox>

Use "" for sdrangel includes and <> for system/library includes.

dforsi commented:
I fixed the wrong include (added by Qt Creator...) and I added missing parts of the help page.

For the Replay tab I reworded the tooltips but I'm not sure that I understood the purpose of the checkbox "Send time to map" because I can change the replay date, but the Cesium map still shows the current time.
